jocktos/src/os.c

```diff
@@ -1,117 +1,102 @@
 /**
-* \brief This module is to...
+* \brief This module is the core JOCKTOS kernel functionality
 */
-
 /* -- Includes ------------------------------------------------------------ */
 // Jocktos
 #include "os.h"
 #include "tcb.h"
 #include "timers.h"
+#include "allocator.h"
 // Middleware
 #include "stm32f303xe.h"

```

Comment:
Know of any way to just stop this error from happening? The linker can still find it so I don't care much. It's just annoying

Reply:
I'm not seeing an error here, are you getting the vscode red squiggle?